sass/utils/_flex.scss

```diff
@@ -4,841 +4,16 @@
     display: flex !important;
 }
 
-.d-inline-flex {
```

Comment:
Wow it's surprising that we still have these in our codebase' it's a good clean up